.travis.yml

```diff
@@ -1,3 +1,5 @@
 language: node_js
 node_js:
   - "node"
+before_script:
```

Comment:
Thinking since we're customizing the build so much at this point we may as well override `script` instead of defining as `before_script`, then move all tasks (including default `test`) into a new `ci` npm script.